ObjectivePGP/PGPPartialKey.m

```diff
@@ -312,8 +312,8 @@ - (nullable PGPSecretKeyPacket *)decryptionPacketForKeyID:(PGPKeyID *)keyID erro
     }
 
     for (PGPPartialSubKey *subKey in self.subKeys) {
-        let signaturePacket = subKey.bindingSignature;
-        if (signaturePacket.canBeUsedToEncrypt && PGPEqualObjects(PGPCast(subKey.primaryKeyPacket, PGPSecretKeyPacket).keyID, keyID)) {
+        //Assume the primary key packet is always capable if if its a secret key packet
+        if (PGPEqualObjects(PGPCast(subKey.primaryKeyPacket, PGPSecretKeyPacket).keyID, keyID)) {
```

Comment:
I'm not sure if this is correct.
This makes always use the first subkey. What if there's multiple subkeys, ale there's one that is valid, and it's not the first from the list?